### ISSUE TITLE
Attaching volumes to droplets fail because of droplet id type mismatch in Python

### ISSUE THREAD

**Issue by darklow** [pulumi/pulumi-digitalocean#218]
Attaching volumes to droplets doesn't work in python and it fails.

## Expected behavior
Attaching volumes do work as in example.

## Current behavior
Fails with an error:
`error: digitalocean:index/volumeAttachment:VolumeAttachment resource 'vol-name' has a problem: Attribute must be a whole number, got 123456789. Examine values at 'VolumeAttachment.DropletId'.`

Reason is quite obvious:
Droplet ID returned by droplet creation is type str - `pulumi.Input[str]`
VolumeAttachment expects Droplet ID to be int - `pulumi.Input[int]`

## Steps to reproduce

Using default example from Volumes docs:
https://www.pulumi.com/docs/reference/pkg/digitalocean/volume/

## Context (Environment)

Python language. Created droplet in step one. Added volume and attachment on step two and tried running `pulumi up`.
pulumi-2.25.2 
pulumi-digitalocean-3.8.0

## Affected feature

VolumeAttachment

**Comment by darklow** [comment]
Found workaround to an error using `droplet.id.apply(int)`:
```py
vol_attach = do.VolumeAttachment("vol-attach",
                     droplet_id=droplet.id.apply(int),
                     volume_id=volume.id)
```

**Comment by infin8x** [comment]
Duplicate of pulumi/pulumi-terraform-bridge#352

We're tracking the larger fix for this issue in the pulumi-terraform-bridge issue mentioned above. For future discoverers of this issue, you can unblock yourself using the workaround: https://github.com/pulumi/pulumi-digitalocean/issues/218#issuecomment-822072747
